### PR TITLE
Merge final release/18.2 to trunk

### PR DESCRIPTION
**Pull request by AliSoftware**
- [x] WPAndroid submitted to Google for review
- [x] JPAndroid submitted to Google for review
- [x] WPAndroid approved by Google and published
- [x] JPAndroid approved by Google and published

### CODE REVIEW

**Review by mokagio — APPROVED** [review]
Good to merge once the app ships. :shipit:

**Comment by AliSoftware** [comment]
FYI, rollout of 18.2 is on hold until we confirm a bug that we talked about last Friday evening on rc3 (ref: p1632125882408200/1631899386.380600-slack-C6UJ0KRKQ) was fixed in Gutenberg 1.16.1 and in the final build – as we think it was – or if we need to hotfix it and postpone the 18.2 rollout.

I'll wait for confirmation about this today before deciding to start the 18.2 rollout (and merge this PR) or not (and put this PR on hold)

**Pull request merged**
